great_expectations/checkpoint/checkpoint.py

```diff
@@ -107,7 +107,7 @@ class Config:
     def _validate_validation_definitions(
         cls, validation_definitions: list[ValidationDefinition] | list[dict]
     ) -> list[ValidationDefinition]:
-        from great_expectations import project_manager
+        from great_expectations.data_context.data_context.context_factory import project_manager
```

Comment:
Not loving the `data_context.data_context.` duplication, but I'm assuming there's a good reason for it, and more importantly it's not part of the public API.

Reply:
I tried to revert this in a bunch and this caused circular dependency issues. I'm guessing we don't always have to do this and we might be able to resolve but I'm going to check it in like this for now and we can investigate later.